src/app/component/login/login.component.scss

```diff
@@ -0,0 +1,21 @@
+
+.login-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100vh;
+
+    form {
```

Comment:
please give on element <form> class name and use in scss 
its more common

Reply:
fixed